docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -1,5 +1,18 @@
 # This document lists known breaking changes in Roslyn after .NET 6 all the way to .NET 7.
 
+## System.TypedReference considered managed
+
+***Introduced in Visual Studio 2022 version 17.6***
+
+Moving forward the `System.TypedReference` type is considered managed.
+
+```csharp
+unsafe
+{
+    TypeReference* r = null; // warning: This takes the address of, gets the size of, or declares a pointer to a managed type
```

Comment:
>TypeReference* r = null; // warning: This takes the address of, gets the size of, or declares a pointer to a managed type [](http://example.com/codeflow?start=4&length=121)

Is this the only possible negative effect? Could this cause an error in some scenarios?

Reply:
Yes, there would also be errors for example with the `unmanaged` constraint.

Reply:
Never mind, we have an error already for TypedReference as a type argument (regardless of constraints). So no other negative effect.